extensions/extensions-helm/aissemble-localstack-chart/templates/credential-secret.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.createCredentialSecret }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: remote-auth-config
```

Comment:
A: I could see it being useful to configure this to a different value, if I decide to use a different name for my CI/cloud secret.

Reply:
Added yaml config option